decoder.rb

```diff
@@ -0,0 +1,75 @@
+MORSE_CODE = {
+  '.-' => 'a',
+  '-...' => 'b',
+  '-.-.' => 'c',
+  '-..' => 'd',
+  '.' => 'e',
+  '..-.' => 'f',
+  '--.' => 'g',
+  '....' => 'h',
+  '..' => 'i',
+  '.---' => 'j',
+  '-.-' => 'k',
+  '.-..' => 'l',
+  '--' => 'm',
+  '-.' => 'n',
+  '---' => 'o',
+  '.--.' => 'p',
+  '--.-' => 'q',
+  '.-.' => 'r',
+  '...' => 's',
+  '-' => 't',
+  '..-' => 'u',
+  '...-' => 'v',
+  '.--' => 'w',
+  '-..-' => 'x',
+  '-.--' => 'y',
+  '--..' => 'z',
+  ' ' => ' ',
+  '.----' => '1',
+  '..---' => '2',
+  '...--' => '3',
+  '....-' => '4',
+  '.....' => '5',
+  '-....' => '6',
+  '--...' => '7',
+  '---..' => '8',
+  '----.' => '9',
+  '-----' => '0'
+}.freeze
+
+# decoding a Morse code character, and return the corresponding character in uppercase
+def decode_morse_character(char)
+  MORSE_CODE[char].upcase
+end
+
+# split sentence to single words
+def split_message(message)
+  message.split('   ')
+end
+
+# decode an entire word in Morse code, takes a string parameter, and return the string representation.
+# Every character in a word will be separated by a single space.
+def decode_word(word)
+  decoded_word = ''
+  word.split.each do |morse|
+    decoded_char = decode_morse_character(morse)
+    decoded_word += decoded_char.to_s
+  end
+  decoded_word
+end
+
+# method to decode the entire message in Morse code, takes a string parameter, and return the string
+# representation. Every word will be separated by 3 spaces.
+def decode(message)
```

Comment:
- [ ] Your program is not producing the right decode  output in the console. Only a single output is printed when your app is run. Kindly compare the attached screenshots and ensure that the full decoded messages is printed in the console.

**Your output:**
![image](https://user-images.githubusercontent.com/12745474/181514137-a18946c1-2a22-4644-8c01-2cd63d105be3.png)


**Expected output:**
![image](https://user-images.githubusercontent.com/12745474/181513908-7aa5215e-1831-46b0-aa4f-26ffa2fe1794.png)

Reply:
This issue is well resolved! ✔️